src/lib/legacy/README.md

```diff
@@ -3,3 +3,5 @@
 This directory includes support for libraries that have been discontinued, but are still relied upon in some part(s) of the repository.
 
 The `/legacy/` libraries are expected to be removed eventually; this is merely a transitory location for the code to remain.
+
+TODO: Move to `/src/lib/public/legacy`.
```

Comment:
Move `/src/lib/legacy` to `/src/lib/public/legacy` (and remove highlighted TODO).